README.md

```diff
@@ -51,22 +51,11 @@ Before start contributing please read the [CONTRIBUTING.md](https://github.com/c
-| <img src="https://github.com/kashyap-shah/Awesome-Web-Art/blob/master/kashyap-shah/loading-animation/Loading_Animation.gif" width=80% height=70%> | [Find the code here](https://github.com/kashyap-shah/Awesome-Web-Art/blob/master/kashyap-shah/loading-animation/animation.html) | [Kashyap Shah](https://github.com/kashyap-shah) |
-| <img src="https://user-images.githubusercontent.com/59108190/94916007-7acc1780-04cb-11eb-94ae-027dd8f3c309.gif" width=80% height=70%> | [Find the code here](https://github.com/SmitShah090/Awesome-Web-Art/blob/master/SmitShah090/Music%20Player/Edu_Hub) | [Smit Shah](https://github.com/SmitShah090) |
-| <img src="https://user-images.githubusercontent.com/61206322/94914206-35f2b180-04c8-11eb-8b39-d8a63f79358a.gif" width=80% height=70%> | [Find the code here](https://github.com/JayMandaviya/Awesome-Web-Art/blob/master/JayMandaviya/button.html) | [Jay Mandaviya](https://github.com/JayMandaviya) |
-| <img src="https://user-images.githubusercontent.com/61206322/94916259-ffb73100-04cb-11eb-92ac-56f9c7bd838c.gif" width=80% height=70%> | [Find the code here](https://github.com/JayMandaviya/Awesome-Web-Art/blob/master/JayMandaviya/cloth.html) | [Jay Mandaviya](https://github.com/JayMandaviya) |
-| <img src="https://user-images.githubusercontent.com/61206322/94917413-1b233b80-04ce-11eb-89fa-64b5ec716555.gif" width=80% height=70%> | [Find the code here](https://github.com/JayMandaviya/Awesome-Web-Art/blob/master/JayMandaviya/Name.html) | [Jay Mandaviya](https://github.com/JayMandaviya) |
-| <img src="https://github.com/kashyap-shah/Awesome-Web-Art/blob/master/kashyap-shah/Snake_Game.gif" width=80% height=70%> | [Find the code here](https://github.com/kashyap-shah/Awesome-Web-Art/tree/master/kashyap-shah/Snake%20Game%20-%20JavaScript) | [Kashyap Shah](https://github.com/kashyap-shah) |
-| <img src="https://github.com/HetGalia/Awesome-Web-Art/blob/master/HetGalia/Pendulum/Pendulum.gif" width=80% height=70%> | [Find the code here](https://github.com/HetGalia/Awesome-Web-Art/tree/master/HetGalia/Pendulum) | [HetGalia](https://github.com/HetGalia) |
-| <img src="https://github.com/kashyap-shah/Awesome-Web-Art/blob/master/kashyap-shah/Weather_App.gif" width=80% height=70%> | [Find the code here](https://github.com/kashyap-shah/Awesome-Web-Art/tree/master/kashyap-shah/Weather%20App%20-%20Javascript%20%26%20API) | [Kashyap Shah](https://github.com/kashyap-shah) |
-| <img src="https://github.com/HetGalia/Awesome-Web-Art/blob/master/HetGalia/tic-tac-toe/Tic%20Tac%20Toe.gif" width=80% height=70%> | [Find the code here](https://github.com/HetGalia/Awesome-Web-Art/tree/master/HetGalia/tic-tac-toe) | [HetGalia](https://github.com/HetGalia) |
+
+
+| <img src="https://github.com/imjp19/Awesome-Web-Art/blob/master/imjp19/HOVER.gif" width=80% height=70%> | [Find the code here](https://github.com/imjp19/Awesome-Web-Art/blob/master/imjp19/button-hover.html) | [Jay Patel](https://github.com/imjp19) |
```

Comment:
Hey @imjp19 what you have added is not in proper aligned table, [check here](https://github.com/clubgamma/Awesome-Web-Art/blob/021a370bcb4fc894b174919031c25f9a8f4b606a/README.md) in the last 2 rows the table is broke.